<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClass</class>
 <widget class="QDialog" name="DialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>授权面板</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>激活</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Author">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>219</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Enter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入激活码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Apply">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Apply">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>本设备授权申请码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Note">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>此设备未授权，请输入激活码授权！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTempAuthority">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>临时授权</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
